### PR TITLE
Fix: Typo

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -636,7 +636,7 @@ If you find yourself creating a lot of [http requests](#http-requests) with the
    class MyTest extends KernelTestCase
    {
        use HasBrowser {
-           browser as baseKernelBrowser;
+           httpBrowser as baseKernelBrowser;
```
Comment (kbond) [inline README.md]: The typo is actually below: 

```diff
-protected function httpBrowser(): KernelBrowser
+protected function browser(): KernelBrowser
```

**Review by kbond — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (OskarStark) [inline README.md]:
```suggestion
           browser as baseKernelBrowser;
```

**Comment by OskarStark** [comment]
Done

**Review by kbond — APPROVED** [review]

**Comment by kbond** [comment]
Thanks @OskarStark!

**Pull request merged**